.travis.yml

```diff
@@ -171,18 +132,18 @@ jobs:
  
  # Clippy linting.
   - name: clippy
-    rust: nightly-2018-10-23
+    rust: nightly-2018-11-28
     install:
-    - cargo clippy --version | rustup component add clippy-preview --toolchain=nightly-2018-10-23
+    - cargo clippy --version || rustup component add clippy-preview --toolchain=nightly-2018-11-28
     - cargo clippy --version
     script:
     - cargo clippy --all -- -D warnings
 
   # Benchmarks (compilation only).
   - name: benchmarks
-    rust: nightly-2018-10-23
+    rust: nightly-2018-11-28
     script:
-    - RUST_LOG=off cargo bench --verbose --manifest-path exonum/Cargo.toml --features long_benchmarks --no-run
+    - RUST_LOG=off cargo check --benches --features "long_benchmarks"
```

Comment:
Just out of curiosity, why have you added quotes around `long_benchmards`. As I know it works without it as well.